Erreur inconnue (veuillez faire un rapport de bug) : <type 'exceptions.AttributeError'>\n
Sa valeur est : AttributeError('instance attr: name',)
  File "<script>", line 9, in <module>
